src/groundStation/system.py

```diff
@@ -1183,7 +1183,23 @@ def __init__(self):
                                 'temp5' : '>b',
                                 'temp6' : '>b',
                                 'temp7' : '>b',
-                                'temp8' : '>b'  
+                                'temp8' : '>b',
+                                #DFGM
+                                '###############################\r\n'
+                                'DFGM Board\r\n'+
+                                '###############################\r\n'+
+                                'Core_Voltage': '>f4',
+                                'Sensor_Temperature': '>f4',
+                                'Reference_Temperature': '>f4',
+                                'Board_Temperature': '>f4',
+                                'Positive_Rail_Voltage': '>f4',
+                                'Input_Voltage': '>f4',
+                                'Reference_Voltage': '>f4',
+                                'Input_Current': '>f4',
+                                'Reserved_1': '>f4',
```

Comment:
What are these? Are they on the satellite side as well? If so could you add them to the housekeeping google sheet?

Reply:
Also I though these were integers

Reply:
That's the DFGM HK data. They should all already be on the google sheet right now aside from the "Reserved" values, which I'll probably add to the sheets tomorrow. Also, you're right! They're supposed to be uint16_t, so I'll change them all from >f4 to >u2 sometime tomorrow